.github/workflows/tox.yml

```diff
@@ -0,0 +1,21 @@
+name: tox
+
+on: pull_request
+
+jobs:
+  lint:
```

Comment:
Need to rename to something else besides lint as the flake8 test also uses lint as an identifier.

Reply:
Could just name it 'unit' imo